deftree/build_deftree_test.go

```diff
@@ -1,27 +1,34 @@
 package deftree
 
 import (
+	"os"
+	"path/filepath"
 	"reflect"
 	"testing"
 
 	// This has to be imported because it modifies the state of `proto` by
 	// registering the `google.api.http` extension, allowing us to specify it
 	// in the sources below.
 	_ "github.com/TuneLab/go-truss/deftree/googlethirdparty"
-	//"github.com/davecgh/go-spew/spew"
 
 	"github.com/golang/protobuf/proto"
 	descriptor "github.com/golang/protobuf/protoc-gen-go/descriptor"
 )
 
+var GOPATH string
+
+func init() {
+	GOPATH = filepath.SplitList(os.Getenv("GOPATH"))[0]
```

Comment:
I just found this: https://github.com/golang/go/blob/master/src/cmd/go/build.go#L1790
```
for _, root := range gopath {
```

I could just range over the `gopath` list. This can be used throughout truss I believe. I can start rolling in this larger change in the morning.